routes/conversations.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Router = require('express').Router;
+const Conversation = require('../models/Conversation');
+const Message = require('../models/Message');
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const router = Router();
+
+
+router.post('/', (req, res) => {
+  let conversation = new Conversation({
+    from_email: req.body.from_email,
+    to_email: req.body.to_email
+  });
+
+  conversation.save((err) => {
+    if (err){
+      res.status(400).json(err);
+    } else {
+      res.status(201).json(conversation);
+    }
+  });
+})
+
+router.get('/:email/messages/send', (req, res) => {
```

Comment:
How is this route used? What does it do?  It's not very clear.  Also, we don't even use the `:email` in the path, so that's already a red flag.